test/system/120-load.bats

```diff
@@ -81,25 +81,89 @@ verify_iid_and_name() {
 @test "podman image scp transfer" {
     skip_if_remote "only applicable under local podman"
 
-    skip "FIXME FIXME FIXME: this needs a big rewrite"
-
-    get_iid_and_name
+    # The testing is the same whether we're root or rootless; all that
+    # differs is the destination (not-me) username.
     if is_rootless; then
+        # Simple: push to root.
         whoami=$(id -un)
-        # FIXME: first, test that we can sudo. If we can't, skip.
-        # FIXME: test 'scp $IMAGE root@localhost::'
-        # FIXME: then test the rest
-        # FIXME: check output
-        run_podman image scp $whoami@localhost::$iid root@localhost::
-        is "$output" "Loaded image.*: $iid" "...."
-
-        # FIXME: "-q" is a NOP
-        run_podman image scp -q $whoami@localhost::$iid root@localhost::
+        notme=root
+        _sudo() { command sudo -n "$@"; }
     else
-        # root
-        # FIXME: identify a rootless user. DO NOT CREATE ONE.
-        run_podman image scp root@localhost::$iid 1000:1000@localhost::
+        # Harder: our CI infrastructure needs to define this & set up the acct
+        whoami=root
+        notme=${PODMAN_ROOTLESS_USER}
+        if [[ -z "$notme" ]]; then
+            skip "To run this test, set PODMAN_ROOTLESS_USER to a safe username"
+        fi
+        _sudo() { command sudo -n -u "$notme" "$@"; }
+    fi
+
+    # If we can't sudo, we can't test.
+    _sudo true || skip "cannot sudo to $notme"
+
+    # FIXME FIXME FIXME: it'd be reeeeeeally nice if we could pass --root
```

Comment:
do you mean change the graph root here? can you not just do `run_podman --root=....` also I have seen some tests that use a different podman call rather than run_podman, could be a solution

Reply:
I meant, some way to have the `sudo`'ed podman have a custom `--root`, so that I, the userid running tests, do not cause harm to a different user. They can't both have the same root, of course; and there's no actual use to scp'ing to/from a different root on the test user. So it would be something like `scp myimage root@localhost[--root=/tmp/fakeroot]` which has no conceivable purpose other than testing. So I should really just delete that comment!